docker/lr-align/Dockerfile

```diff
@@ -55,3 +60,10 @@ RUN wget https://github.com/samtools/samtools/releases/download/1.11/samtools-1.
     && make \
     && make install
 
+# install gatk
+RUN git clone https://github.com/broadinstitute/gatk.git -b kvg_pbeap \
```

Comment:
can you explain why installing it this way?
I'm trying to reduce our dependency on non-standard GATK as much as possible.